sim/mage/arcane/arcane.go

```diff
@@ -31,13 +37,9 @@ func NewArcaneMage(character *core.Character, options *proto.Player) *ArcaneMage
 	}
 	arcaneMage.ArcaneOptions = arcaneOptions
 
-	return arcaneMage
-}
-
-type ArcaneMage struct {
-	*mage.Mage
+	arcaneMage.ApplyArcaneSpecInnate()
```

Comment:
This should not be called from the NewArcaneMage func but from the initialize instead